extensions/applicationinsights-dependencies-js/Tests/Unit/src/ajax.tests.ts

```diff
@@ -1628,6 +1628,124 @@ export class AjaxTests extends AITestClass {
             }]
         });
 
+        this.testCaseAsync({
```

Comment:
I cloned the two test cases above this that were testing for `null`/`undefined`, but I didn't completely understand what all of the setup/checks were that were necessary.